strawberry/types/info.py

```diff
@@ -36,15 +45,15 @@ def schema(self) -> "Schema":
         return self._raw_info.schema._strawberry_schema  # type: ignore
 
     @property
-    def field_nodes(self) -> List[FieldNode]:  # deprecated
+    def field_nodes(self) -> Collection[FieldNode]:  # deprecated
         warnings.warn(
             "`info.field_nodes` is deprecated, use `selected_fields` instead",
             DeprecationWarning,
         )
         return self._raw_info.field_nodes
 
     @cached_property
-    def selected_fields(self) -> List[Selection]:
+    def selected_fields(self) -> Collection[Selection]:
```

Comment:
```suggestion
    def selected_fields(self) -> List[Selection]:
```

Our `convert_selections` function returns `List[Selection]`, so we could continue using `List[Selection]` here.

https://github.com/strawberry-graphql/strawberry/blob/b50c87d7d53b4a099151101889d69a4d23c440d5/strawberry/types/nodes.py#L64-L79